siem/graylog.conf

```diff
@@ -0,0 +1,736 @@
+############################
```

Comment:
It appears most of this is commented out. While on a normal system it would be beneficial to keep the whole file intact for future changes, in a Docker container it's helpful to make it obvious what is being changed. Can we just include the actual configuration parameters being changes (and applicable comments) and remove the rest? Can you also comment on why the default configuration wasn't suitable?

Reply:
This setting is desired to stop Graylog flooding logs on Elastic search:
elasticsearch_mute_deprecation_warnings = true